tests/apm/serverHeartbeatSucceededEvent-001.phpt

```diff
@@ -68,6 +68,8 @@ object(MongoDB\Driver\Monitoring\ServerHeartbeatSucceededEvent)#%d (%d) {
   int(%d)
   ["awaited"]=>
   bool(%s)
+  ["durationMicros"]=>
+  int(%d)
```

Comment:
I understand that this value was already exposed with an [`getDurationMicros`](https://www.php.net/manual/en/class.mongodb-driver-monitoring-serverheartbeatsucceededevent.php) method, but now it becomes accessible with var_dump.

Reply:
That's correct :)